heartbeat/IPaddr2

```diff
@@ -1002,6 +1003,11 @@ ip_served() {
 
 	if [ -z "$IP_CIP" ]; then
 		for i in $cur_nic; do
+                        # check address label
+                        if [ ! -z $IFLABEL -a -z "`$IP2UTIL -o -f $FAMILY addr show $nic label $IFLABEL`" ]; then
```

Comment:
We avoid -a/-o where possible: https://web.archive.org/web/20230306130858/https://wiki.bash-hackers.org/commands/classictest#why_you_should_avoid_using_-a_and_-o

Can you update it to `[ ! -z $IFLABEL ] && [ -z "`$IP2UTIL -o -f $FAMILY addr show $nic label $IFLABEL`" ]`?

Reply:
Done!

Reply:
Oops. I shouldnt have used single backquotes for formatting, which made them cancelled out in the comment.

Can you add the backquotes or $( ) around the command again?

Reply:
We ended up with some additional improvements based on your fix, so no need to update this PR.

I'll just make a new PR, and give you credit there.